server/lib/budget.js

```diff
@@ -105,19 +105,31 @@ export function getTotalAmountReceivedAmount(collective, { startDate, endDate, c
   });
 }
 
-export function getTotalNetAmountReceivedAmount(collective, { startDate, endDate, currency, version } = {}) {
+export async function getTotalNetAmountReceivedAmount(collective, { startDate, endDate, currency, version } = {}) {
   version = version || collective.settings?.budget?.version || 'v1';
   currency = currency || collective.currency;
-  return sumCollectiveTransactions(collective, {
+
+  const totalReceived = await sumCollectiveTransactions(collective, {
     startDate,
     endDate,
     currency,
     column: ['v0', 'v1'].includes(version) ? 'netAmountInCollectiveCurrency' : 'netAmountInHostCurrency',
-    kind: ['CONTRIBUTION', 'ADDED_FUNDS', 'HOST_FEE', 'PAYMENT_PROCESSOR_FEE'],
-    isDebt: false,
+    transactionType: CREDIT,
     hostCollectiveId: version === 'v3' ? { [Op.not]: null } : null,
     excludeInternals: true,
   });
+
+  const totalFees = await sumCollectiveTransactions(collective, {
+    startDate,
+    endDate,
+    currency,
+    column: ['v0', 'v1'].includes(version) ? 'netAmountInCollectiveCurrency' : 'netAmountInHostCurrency',
+    transactionType: DEBIT,
+    kind: ['HOST_FEE'],
```

Comment:
Maybe better to refer this string from,

https://github.com/opencollective/opencollective-api/blob/7dacf7cdc7f7b45b5a9cc01149589e62630bb8ab/server/constants/transaction-kind.ts#L9 

so that we have a single place where we can change things in the future. 😄

Reply:
Correct, it's good to use the enums when they exist